packs/equipment-effects/effect-whispering-staff-enemy.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Whispering Staff (Enemy)",
     "system": {
         "description": {
-            "value": "<p>Through the staff's strange whispering, you gain a glimpse into the mind and desires of one creature you can see within 30 feet. Until the end of your next turn, that creature is @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] to you and takes a –2 circumstance penalty to saving throws against your spells.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Whispering Staff]</p>\n<p>You are take a –2 circumstance penalty to saving throws against spells.</p>"
```

Comment:
Consider updating the selector to `"selector": "saving-throw"`.

Also, the "your spells" part could be handled with a signature predicate.

Reply:
Check when you get a moment to see if this works.

Reply:
> Check when you get a moment to see if this works.

It looks reasonable. Did you test?

Reply:
I did. 

Sorry, bad habit from work. I always ask people to double-check as a sanity check.